### PR TITLE
[VMD] Add contentDescription to image buttons

### PR DESCRIPTION
**Pull request by npresseault**
## Description
This adds contentDescription field to `VMDImageContent` and others

## Motivation and Context
Needed for accessibility, in particular for VMDButtons with images.

## How Has This Been Tested?
Tested via sample

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

The change is not binary compatible due to default arguments and data classes being used for `VMDImageContent` and others, it is, however, source compatible.

### CODE REVIEW

**Review by Boubalou — COMMENTED** [review]
Would it be better to not include changes to the versions used everywhere (ref 5.2.0 in sample, etc...) in an actual feature? They are kind of 2 different things to be honest.

File: trikot-kword/sample/ios/Podfile.lock [hunk 1]
```diff
@@ -21,9 +21,9 @@ EXTERNAL SOURCES:
...
   SwiftLint: 99f82d07b837b942dd563c668de129a03fc3fb52
-  Trikot: a5b1e56254b07569577503f4f94613b388d087ab
-  TRIKOT_FRAMEWORK_NAME: 60d9a2c5a48c549e3799233bcd5a73ea483c4eb8
+  Trikot: 80b00dc0b6f8e05846893d73fac9b9e55348fd1e
+  TRIKOT_FRAMEWORK_NAME: 67700accc0c3e39aaddb1edf24d25fd972a27e08
 
 PODFILE CHECKSUM: cd5b1ad5ff7d13e99dec5f4c7a746099a745e482
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.11.2
```
Comment (CharlesMcD) [inline trikot-kword/sample/ios/Podfile.lock:29]: On pourrait pas monter à 1.13.0 au lieu de descendre à 1.11.2?

**Comment by npresseault** [comment]
@Boubalou Version changes in sample are not really impactful, at this point it's just going to be more hassle to open a different PR. Will take note for next time though ;)

**Review by Boubalou — COMMENTED** [review]
Late 👍

**Pull request merged**